types/k6/experimental/browser.d.ts

```diff
@@ -772,6 +777,96 @@ export interface BrowserContext {
     ): Page | null;
 }
 
+
+/**
+ * {@link Cookie} represents a cookie in a {@link BrowserContext}.
+ *
+ * @see
+ * {@link BrowserContext} has methods to {@link BrowserContext.addCookies | add}, {@link BrowserContext.cookies | query} and {@link BrowserContext.clearCookies | clear} cookies.
+ */
+export interface Cookie {
+    /**
+     * The {@link Cookie | cookie}'s name.
+     *
+     * @defaultValue
+     * The default is `''`.
+     */
+    name: string;
+
+    /**
+     * The {@link Cookie | cookie}'s value.
+     *
+     * @defaultValue
+     * The default is `''`.
+     */
+    value: string;
+
+    /**
+     * The {@link Cookie | cookie}'s URL.
+     *
+     * Required unless one of {@link Cookie.domain | domain} or {@link Cookie.path | path} are specified.
+     */
+    url?: string;
+
+    /**
+     * The {@link Cookie | cookie}'s domain.
+     *
+     * Required unless one of {@link Cookie.url | url} or {@link Cookie.path | path} are specified.
+     */
+    domain?: string;
+
+    /**
+     * The {@link Cookie | cookie}'s path.
+     *
+     * Required unless one of {@link Cookie.url | url} or {@link Cookie.domain | domain} are specified.
+     */
+    path?: string;
```

Comment:
According to our docs, `path` defaults to `/`. Should we add that here too?

Reply:
Makes sense! See: [58003a0](https://github.com/grafana/k6-DefinitelyTyped/pull/14/commits/58003a099e79783798c9e776a49fecb5f741dbfd)